packages/react-component-library/src/components/SelectBase/SelectBaseOption.tsx

```diff
@@ -14,20 +14,10 @@ export interface SelectBaseOptionBaseProps extends ComponentWithClass {
   title?: string
 }
 
-export interface SelectBaseOptionAsArrayProps
-  extends SelectBaseOptionBaseProps {
-  children: React.ReactNodeArray
+export interface SelectBaseOptionProps extends SelectBaseOptionBaseProps {
+  children: React.ReactNode
```

Comment:
The union type of not needed here for children. The type ReactNode contains `string` and `Iterable<React.ReactNode>`
```ReactNode == string | number | boolean | {} | React.ReactElement<any, string | React.JSXElementConstructor<any>> | Iterable<React.ReactNode> | React.ReactPortal | null | undefined```